src/gamestate/system_state.cpp

```diff
@@ -281,7 +281,7 @@ void state::on_mouse_wheel(int32_t x, int32_t y, key_modifiers mod, float amount
 		}
 	}
 }
-void state::on_key_down(virtual_key keycode, key_modifiers mod) {
+void state::on_key_down(virtual_key keycode, key_modifiers mod, bool can_move_map_while_visible) {
```

Comment:
I don't see why this needs to be a parameter if it is already a member variable of the state object